homeassistant/components/awair/const.py

```diff
@@ -39,7 +43,7 @@
 
 DOMAIN = "awair"
 
-DUST_ALIASES = [API_PM25, API_PM10]
+DUST_ALIASES = [API_PM25, API_PM10, API_PM10EST]
```

Comment:
Gen1 Awair devices had a "dust" sensor that reported combined pm10+pm2.5 values. This array was used [elsewhere](https://github.com/home-assistant/core/blob/178394e09e5e2f8de73e53aee7b8cda36a21b197/homeassistant/components/awair/sensor.py#L57-L69) to turn the "dust" sensor reading into identical pm2.5 and pm10 sensors, which felt more in line with the rest of homeassistant at that time.

Since the Awair cloud API is no more, we don't actually have to update this array. A future PR could remove that aliasing entirely.